src/components/molecules/StartTable/StartTable.tsx

```diff
@@ -1,18 +1,18 @@
 import React from "react";
```

Comment:
[Q] Should this component be moved to `./src/components/attendee` or is it going to be only temporary until proper attendee component gets created?